### ISSUE TITLE
path parameters function doesn't seem to capture missing path parameters in operations

### ISSUE THREAD

**Issue by TristanSpeakEasy** [daveshanley/vacuum#207]
This doc:

```yaml
openapi: 3.0.3
info:
  title: Test
  version: 0.0.1
  description: A test API
servers:
  - url: https://example.com
security:
  - basicAuth: []
tags:
  - name: tag1
paths:
  /update/{something}:
    post:
      operationId: postSomething
      summary: Post something
      tags:
        - tag1
      responses:
        '200':
          description: Post OK
    get:
      operationId: getSomething
      summary: Get something
      tags:
        - tag1
      responses:
        '200':
          description: Get OK
components:
  securitySchemes:
    basicAuth:
      type: http
      scheme: basic
```

returns this result from vacuum:

```
vacuum lint ../openapi-specs/path-param-test.yaml -d

INFO  Linting against 42 rules: https://quobix.com/vacuum/rulesets/recommended

Linting passed, A perfect score! well done!
```

looking at the code I can see there is a check to detect the fact there is no path param defined for {something} but it doesn't seem to be catching that it isn't defined

**Comment by daveshanley** [comment]
Hmm.. i'll look into this asap.

**Comment by daveshanley** [comment]
This should be fixed in `v0.0.44`

**Comment by TristanSpeakEasy** [comment]
can confirm it is fixed
